src/endpoint/s3/s3_utils.js

```diff
@@ -271,6 +272,17 @@ function set_response_object_md(res, object_md) {
     if (storage_class !== STORAGE_CLASS_STANDARD) {
         res.setHeader('x-amz-storage-class', storage_class);
     }
+    if (object_md.restore_status) {
+        const restore = [`ongoing-request="${object_md.restore_status.ongoing}"`];
+        if (!object_md.restore_status.ongoing) {
+            // Expiry time looks like "Fri, 21 Dec 2012 00:00:00 GMT"
+            const expiry_date = moment(object_md.restore_status.expiry_time).utc().format('ddd, DD MMM YYYY HH:mm:ss [GMT]');
```

Comment:
I think we usually use new Date(object_md.restore_status.expiry_time).toUTCString()